spec/stats/kbuild/02

```diff
@@ -0,0 +1,378 @@
+2024-02-26 06:47:23 cd /lkp/benchmarks/kbuild/kbuild/1
+2024-02-26 06:47:23 make mrproper
+2024-02-26 06:47:27 make defconfig
+  HOSTCC  scripts/basic/fixdep
+  HOSTCC  scripts/kconfig/conf.o
+  HOSTCC  scripts/kconfig/confdata.o
+  HOSTCC  scripts/kconfig/expr.o
+  LEX     scripts/kconfig/lexer.lex.c
+  YACC    scripts/kconfig/parser.tab.[ch]
+  HOSTCC  scripts/kconfig/lexer.lex.o
+  HOSTCC  scripts/kconfig/parser.tab.o
+  HOSTCC  scripts/kconfig/preprocess.o
+  HOSTCC  scripts/kconfig/symbol.o
+  HOSTLD  scripts/kconfig/conf
+*** Default configuration is based on 'x86_64_defconfig'
+#
+# configuration written to .config
+#
+2024-02-26 06:47:29 scripts/config --file .config --disable CONFIG_UNUSED_SYMBOLS
+2024-02-26 06:47:29 scripts/config --file .config --enable CONFIG_TRIM_UNUSED_KSYMS
+start time: 1708930050.011573855
+
+real: 33.544s
+user: 910.672s
+sys: 160.023s
+end time: 1708930083.557520310
+2024-02-26 06:48:03 make mrproper
+  CLEAN   arch/x86/entry/vdso
+  CLEAN   arch/x86/kernel/cpu
+  CLEAN   arch/x86/kernel
+  CLEAN   arch/x86/realmode/rm
+  CLEAN   arch/x86/lib
+  CLEAN   certs
+  CLEAN   drivers/firmware/efi/libstub
+  CLEAN   drivers/scsi
+  CLEAN   drivers/tty/vt
+  CLEAN   drivers/video/logo
+  CLEAN   lib
+  CLEAN   net/wireless
+  CLEAN   security/selinux
+  CLEAN   usr
+  CLEAN   arch/x86/tools
+  CLEAN   include/ksym
+  CLEAN   modules.builtin.modinfo
+  CLEAN   scripts/basic
+  CLEAN   scripts/kconfig
+  CLEAN   scripts/mod
+  CLEAN   scripts/selinux/genheaders
+  CLEAN   scripts/selinux/mdp
+  CLEAN   scripts
+  CLEAN   include/config include/generated arch/x86/include/generated
+  CLEAN   .config .version Module.symvers
```

Comment:
for above CLEAN lines (from line 28 to line 52), pls only leave one line which is enough for the spec.

Reply:
ok